mnemonic/src/Cardano/Mnemonic/Util.hs

```diff
@@ -0,0 +1,79 @@
+-- | Module for small utility functions.
+module Cardano.Mnemonic.Util
+       ( -- * String manipulation
+         headToLower
+       , stripFieldPrefix
+       , mkJsonKey
+
+       -- * Miscellaneous
+       , eitherToParser
```

Comment:
Don't we only need this `eitherToParser` function here? Rest can still live on the wallet where it's used.